aaq/views.py

```diff
@@ -116,3 +117,37 @@ def check_urgency(request, *args, **kwargs):
 
     return_data = json_msg
     return Response(return_data, status=status.HTTP_202_ACCEPTED)
+
+
+@api_view(("POST",))
+@renderer_classes((JSONRenderer,))
+def search(request, *args, **kwargs):
+    serializer = SearchSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+    query_text = serializer.validated_data["query_text"]
+    generate_llm_response = serializer.validated_data["generate_llm_response"]
+    query_metadata = serializer.validated_data["query_metadata"]
+    url = urllib.parse.urljoin(settings.AAQ_V2_API_URL, "search")
+    payload = {
+        "query_text": f"{query_text}",
+        "generate_llm_response": f"{generate_llm_response}",
+        "query_metadata": f"{query_metadata}",
+    }
+    headers = {
+        "Authorization": settings.AAQ_V2_AUTH,
+        "Content-Type": "application/json",
+    }
+
+    response = requests.request("POST", url, json=payload, headers=headers)
+
+    json_msg = {
```

Comment:
This looks like its just recreating the dict exactly how we got it from AAQ and returning it?

Shouldn't we build a message here like AAQ v1?

Reply:
@Hlamallama Have have you seen this feedback?

Reply:
@DevChima yes, i've implemeted the feedback from Eric